### PR TITLE
JWT backend

### PR DESCRIPTION
**Pull request by danacastroman**

### CODE REVIEW

File: server/helpers/generate-jwt.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import jwt from 'jsonwebtoken';
+
+const generateJWT = (uid: string) => {
+  return new Promise((resolve, reject) => {
+    const payload = {
+      uid
+    };
+
+    jwt.sign(payload, process.env.SECRETORPRIVATEKEY || '', {
+      expiresIn: '24h'
```
Comment (alfonsoLeandro) [inline server/helpers/generate-jwt.ts]: 24 hours might be too much, how about we lower this to 30m or something like that?

**Comment by alfonsoLeandro** [comment]
Looks so good overall, seems you are really getting along with node

File: server/helpers/generate-jwt.ts (hunk 1 above)
Comment (danacastroman) [reply server/helpers/generate-jwt.ts]: I think 30m is too little for our app that does not require much security, for example, like an application for a bank. If you like I can change to 1 hour.

**Review by alfonsoLeandro — APPROVED** [review]
LGMT

**Pull request merged**